Components/badges/badges.css

```diff
@@ -0,0 +1,285 @@
+	color: rgb(37, 35, 35);
+	border: 1.5px solid white;
+}
+
+/* icon-number badges */
+
+.icon-badges {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	justify-items: center;
+	padding: 1.5rem;
+}
+
+.pop-icon {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	justify-items: center;
+}
+
+.icon-badge-container {
+	margin: 15px 26px;
+	position: relative;
+	display: inline-block;
+}
+
+.icon-lg {
+	width: 8rem;
+	height: 8rem;
+}
+
+.icon-md {
+	width: 5rem;
+	height: 5rem;
+}
+
+.icon-sm {
+	width: 3rem;
+	height: 3rem;
+}
+
+.status-number-email {
+	position: absolute;
+	top: 1.5rem;
+	right: 0.3rem;
+	height: 1.2rem;
+	width: 1.2rem;
+	border-radius: 50%;
+	background-color: rgb(189, 59, 8);
+	color: rgb(37, 35, 35);
+	border: 1.5px solid white;
+	text-align: center;
+	cursor: pointer;
+}
+
+.status-number-cart {
+	position: absolute;
+	top: 0.3rem;
+	right: -0.4rem;
+	height: 1rem;
+	width: 1rem;
+	border-radius: 50%;
+	background-color: rgb(202, 15, 62);
+	color: rgb(37, 35, 35);
+	border: 1.5px solid white;
+	text-align: center;
+	cursor: pointer;
+}
+
+.status-number-menu {
+	position: absolute;
+	top: 0rem;
+	right: -0.2rem;
+	height: 1rem;
+	width: 0.9rem;
+	border-radius: 50%;
+	background-color: rgb(55, 0, 128);
+	color: rgb(27, 201, 192);
+	border: 1.5px solid white;
+	text-align: center;
+	cursor: pointer;
+}
+
+/* text-badges */
+
+.txt-badges {
+	display: flex;
+
+	justify-content: center;
+	align-items: center;
+	justify-items: center;
+	flex-direction: column;
+	padding: 1.5rem;
+}
+
+.pop-txt {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	justify-items: center;
+}
+
+.txt-badge-container {
+	margin: 15px 26px;
+	position: relative;
+	display: flex;
+	flex-direction: row;
+}
+
+.txt {
+	line-height: 10%;
+}
+
+.status-txt-head {
+	position: absolute;
+	top: 0.6em;
+	left: 15em;
+	background-color: rgb(55, 0, 128);
+	color: rgb(27, 201, 192);
+	border: 1.5px solid rgb(55, 0, 128);
+	text-align: center;
+	cursor: pointer;
+	border-radius: 20%;
+	height: 1.4rem;
+	width: 2.5rem;
+	font-weight: 600;
+}
+
+.status-txt-p0 {
+	position: absolute;
+	top: 0.8em;
+	left: 13em;
+	background-color: rgb(163, 152, 179);
+	color: rgb(7, 87, 83);
+	border: 1.5px solid rgb(55, 0, 128);
+	text-align: center;
+	cursor: pointer;
+	border-radius: 10%;
+	height: 1.4rem;
+	width: 2.5rem;
+	font-weight: 600;
+}
+
+.status-txt-p0:hover {
+	filter: contrast(40%);
+	background-color: rgb(138, 79, 226);
+	color: rgb(28, 31, 30);
+}
+
+.status-txt-p1 {
+	position: absolute;
+	top: 0.3em;
+	left: 14.5em;
+	background-color: rgb(131, 67, 226);
+	color: rgb(35, 7, 87);
+	text-align: center;
+	cursor: pointer;
+	height: 1.3rem;
+	width: 2.5rem;
+	padding: 3px 6px;
+	font-weight: 600;
+	border-radius: 12rem;
+}
+
+.bad {
+	background-color: rgb(48, 45, 51);
+	color: rgb(216, 206, 233);
+}
+
+.good {
+	background-color: rgb(131, 67, 226);
```

Comment:
for color use color variable and use only one type of color like - hex code or rgba only.